conf/dataset/segmentation/s3dis/s3dis1x1.yaml

```diff
@@ -1,6 +1,7 @@
 # @package dataset
 defaults:
+  - dataset_s3dis
```

Comment:
for `ConfigStore` we have to add a line to all the defaults lines so that it can associate the config file with the dataclass.